frontend/integration-tests/tests/devconsole/git-import-flow.scenario.ts

```diff
@@ -0,0 +1,56 @@
+import { browser, $, element, by, ExpectedConditions as until } from 'protractor';
+import { appHost, checkLogs, checkErrors } from '../../protractor.conf';
+import {
+  navigateImportFromGit,
+  setBuilderImage,
+  enterGitRepoUrl,
+  addApplication,
+  applicationName,
+  appName,
+  builderImage,
+  buildImageVersion,
+} from '../../views/devconsole-view/git-imort-flow';
+import { newApplicationName, newAppName } from '../../views/devconsole-view/new-app-name.view';
+import { switchPerspective, Perspective } from '../../views/devconsole-view/dev-perspective.view';
+import { sideHeader } from '../../tests/devconsole/dev-perspective.scenario';
+
+export const version = element(by.id('8-RHOAR-link'));
+
+describe('git import flow', () => {
+  let newApplication;
+  let newApp;
+  const importFromGitHeader = element(by.id('resource-title'));
+  const createButton = $('.pf-c-button.pf-m-primary');
```

Comment:
Can you add data-test-id to create button instead of rallying on the class name?